### PR TITLE
20 create user registration and login forms

### PR DESCRIPTION
**Pull request by bravint**

### CODE REVIEW

File: src/client/component/Login.jsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import { useState } from 'react';
+
+const Login = () => {
+    const intitalForm = {
```
Comment (polarbear23) [inline src/client/component/Login.jsx]: spelling mistake

File: src/client/components/Register.jsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+        password: '',
+    };
+
+    const [form, setForm] = useState(initialForm);
+
+    const handleChange = (event) => {
+        const { name, value } = event.target;
+
+        setForm({ ...form, [name]: value });
+    };
```
Comment (nathanSamosa) [inline src/client/components/Register.jsx:22]: very clean

File: src/client/components/Welcome.jsx [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+                <h2 className="welcome-heading subtitle">
+                    Or view a leaderboard you have an invite to:
+                </h2>
+                <SearchLeaderboard />
+                {formToRender === 'register' && (
+                    <Register
+                        setFormToRender={setFormToRender}
+                        setIsLoggedIn={setIsLoggedIn}
+                    />
+                )}
```
Comment (nathanSamosa) [inline src/client/components/Welcome.jsx:56]: well done

**Review by nathanSamosa — APPROVED** [review]

**Review by polarbear23 — APPROVED** [review]

**Review by molloyconor511 — APPROVED** [review]

File: .eslintrc.json [hunk 4]
```diff
@@ -19,6 +19,7 @@
         "react"
     ],
     "rules": {
+      "react/prop-types": "off"
```
Comment (glowkeeper) [inline .eslintrc.json:22]: Why turn that off?

File: src/client/components/styling/login.css [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+.login-aside-text {
+    color: var(--clr-white);
+}
+.login-aside-title {
+    font-size: 3rem;
+}
+
+.login-button {
+    height: 2.5rem;
```
Comment (glowkeeper) [inline src/client/components/styling/login.css:37]: these are the sort of values you could create variables for - doing so would help you achieve consistency, i.e. all buttons would be the same width and height (for instance)

**Review by glowkeeper — APPROVED** [review]
Approved (with comments ;) )

File: src/client/components/styling/login.css (hunk 5 above)
Comment (bravint) [reply src/client/components/styling/login.css:37]: We have discussed refactoring the login and register CSS files, to get rid of the repetition in both files. We will change as requested

File: .eslintrc.json (hunk 4 above)
Comment (bravint) [reply .eslintrc.json:22]: It threw errors everytime we tried to de-structure props.

**Pull request merged**